.github/workflows/run-checker-daily-sctp.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+name: Run-checker daily sctp
+on: [pull_request, push]
+# Jobs run daily
+#on:
+#  schedule:
+#    - cron: '0 6 * * *'
+permissions:
+  contents: read
+env:
+    OK:1
+
+jobs:
+  run-checker:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Dependencies for sctp option
+      run:  |
+        sudo apt-get update
+        sudo apt-get -yq install lksctp-tools libsctp-dev
+        checksctp
```

Comment:
This should be enough and get rid of the OK environment variable:
```yaml
- name: Check SCTP is available
  run: checksctp
```

Reply:
Except for when it fails again on merge and I have to spend another day looking at it :)

Reply:
Yeah, that.  This step will fail if checksctp does and the rest won't run.  It avoid the conditionals.

Reply:
We might have to try this out by merging..